pyrasterframes/src/main/python/pyrasterframes/__init__.py

```diff
@@ -167,10 +173,32 @@ def temp_name():
         .load(path, **options)
 
 
-def _aliased_writer(df_writer, format_key, path, **options):
-    """ Saves the dataframe to a file of the given type at the given path."""
-    return df_writer.format(format_key).save(path=path, options=options)
+def _geotiff_writer(
+    df_writer,
+    path=None,
+    crs=None,
+    raster_dimensions=None,
+    **options):
+
+    if raster_dimensions is not None:
+        if isinstance(raster_dimensions, tuple):
+            options.update({
+                "imageWidth": raster_dimensions[0],
+                "imageHeight": raster_dimensions[1]
+            })
+        elif isinstance(raster_dimensions, str):
+            parts = raster_dimensions.split(',')
+            options.update({
+                "imageWidth": parts[0],
+                "imageHeight": parts[1]
+            })
+
+    if crs is not None:
```

Comment:
in our discussions before i thought crs was required?

Reply:
Yea. 

```
Py4JJavaError: An error occurred while calling o107.save.
: java.lang.IllegalArgumentException: requirement failed: A destination CRS must be provided
	at scala.Predef$.require(Predef.scala:224)
```

So we could have a check like this in place of the if statement. 

```python
```suggestion
    assert(crs is not None, "You must pass in a `crs` argument")
```

 WOuld be good in docs to give some details about what's valid. EPSG codes? proj strings? wkt? What about pyproj [CRS object](https://pyproj4.github.io/pyproj/stable/api/crs.html)?  rasterio [CRS](https://rasterio.readthedocs.io/en/stable/api/rasterio.crs.html#rasterio.crs.CRS) Possibly some of those can come later in a future issue / PR.

Reply:
> in our discussions before i thought crs was required?

The only case where it isn't is if it's already a "layer".... probably not the regular use case however.